flags.css

```diff
@@ -1,5 +1,14 @@
 /* Indonesia */
-
+.flagContainer {
+    width: 50vw;
```

Comment:
you might want to make them based on vmin that way the flag remains proportional.

like
```
height: 50vmin;
width: 33vmin;
```